control_msgs/msg/JointJog.msg

```diff
@@ -0,0 +1,18 @@
+# Used in time-stamping the message.
+Header header
+ 
+# Name list of the joints. You don't need to specify all joints of the
+# robot. Joint names are case-sensitive.
+string[] joint_names
+ 
+# Relative displacement of the joints listed in joint_names.
+# The order must be identical.
+# If positions, units are rad for revolute joints, meters for linear joints.
+# If velocities, units are rad/s or m/s.
+float64[] deltas
+
+# Time information may be required.
+# If populated, should be > 0.
+# For position deltas, may indicate the time to reach new position.
+# For velocity deltas, may indicate how long the command is valid.
```

Comment:
A velocity delta would be the acceleration.

Reply:
OK, will address this.